packages/@ama-mfe/ng-utils/src/navigation/restore-route.pipe.ts

```diff
@@ -49,6 +49,7 @@ export class RestoreRoute implements PipeTransform {
   private readonly activeRoute = inject(ActivatedRoute);
   private readonly domSanitizer = inject(DomSanitizer);
   private readonly routeMemorizeService = inject(RouteMemorizeService, { optional: true });
+  private readonly window: Window = inject(Window, { optional: true }) || window;
```

Comment:
Does the explicit typing mandatory?


```suggestion
  private readonly window = inject(Window, { optional: true }) || window;
```